v2/go.mod

```diff
@@ -2,4 +2,7 @@ module github.com/nats-io/jwt/v2
 
 go 1.16
 
-require github.com/nats-io/nkeys v0.3.0
+require (
+	github.com/nats-io/jwt v1.2.2
```

Comment:
Should not have changed this since this jwt dependency had a CVE which makes the project be flagged and the `go_test.mod` would have covered it, I think to avoid this we should release the v1.3.0 without the CVE